packages/bot/src/services/parseSetupMessage.ts

```diff
@@ -2,10 +2,17 @@ import { Message } from 'discord.js'
 
 // parse received setup message
 export function parseSetupMessage (message: Message) {
-  const chunks = message.content.replace('  ', ' ').split(' ')
+  // const chunks = message.content.replace('  ', ' ').split(' ')
   const guildId = message.guild?.id
-  const daoName = chunks[1]
-  const roleAllowed = chunks.slice(2, chunks.length).join(' ')
+  // Split between the `!setup` part, the DAO name, and the allowed role
+  const chunks = message.content.replace('  ', ' ').split(/\@|!setup/)
+    ? message.content.replace('  ', ' ').split(/\@|!setup/)
+    : message.content.replace('  ', ' ').split(/\<@|!setup/)
```

Comment:
I think this can be simplified using the optional marker `?`after the `<` so that we can match either `@` or `<@` :
```suggestion
  const chunks = message.content.replace('  ', ' ').split(/\^?\@|!setup/)
```